index.bs

```diff
@@ -4106,7 +4106,7 @@ partial dictionary MLOpSupportLimits {
 <div dfn-for="MLGraphBuilder/dequantizeLinear(input, scale, zeroPoint, options)" dfn-type=argument>
     **Arguments:**
         - <dfn>input</dfn>: an {{MLOperand}}. The input tensor.
-        - <dfn>scale</dfn>: an {{MLOperand}}. The scale tensor to multiply each input value by after adjusting by the zero point. It must be [=blockwise broadcastable=] with the input.
+        - <dfn>scale</dfn>: an {{MLOperand}}. The scale tensor to multiply each input value by after adjusting by the zero point. It must be [=blockwise broadcastable=] with the input. It must be positive.
```

Comment:
> It must be positive.

What about zero, as zero isn't really positive or negative 🤔. I'd say "Negative scales are unsupported", if we're really concerned about negative values, or if zero was to be disallowed too, then I'd clearly say "They must be positive nonzero values".